odoo/tools/mail.py

```diff
@@ -631,10 +632,14 @@ def encapsulate_email(old_email, new_email):
     if not new_email_split or not new_email_split[0]:
         return
 
-    if old_email_split[0][0]:
-        name_part = '%s (%s)' % old_email_split[0]
+    old_name, old_email = old_email_split[0]
+    if old_name:
+        if include_email:
+            name_part = '%s (%s)' % (old_name, old_email)
+        else:
+            name_part = old_name
     else:
-        name_part = old_email_split[0][1]
+        name_part = old_email
```

Comment:
Isn't that going to trigger the same spoofing alert issue? You'll have an email as name part and a different email as actual email.

Reply:
I also think, it would be cool to replace `@` by `[at]` or something like this

Reply:
We could replace the '@', but that may still look rather fishy, for smarter email providers. Maybe we just drop the domain and keep the local part? Better than nothing?